### ISSUE TITLE
Set up the Home screen - Rooms and Contacts lists

### ISSUE THREAD

**Issue by giomfo** [tchapgouv/tchap-android-v2#5]
We will restore the 2 tabs of the current Tchap: "Conversations " and "Contacts".

We will keep the bottom bar from Element-Android, but change the content of this tab:
- [x] the first tab is "Conversations": the complete list of the rooms ("salons" and "discussion" (= the direct))
CAUTION: the discussions created to invite a contact by email are not listed in this rooms list until the invite is pending. This point will be managed by #22 
- [x] the second one is "Contacts" composed by 2 sections
   - [x] first section "Contacts Tchap" which merges 2 kinds of contacts:
      -  [x] the contacts of the phone book who have a Tchap account (we consider here only the contacts with an email address. If a contact has several emails addresses, we duplicate the contact. This eases the contact handling (only one email by contacts). I think we will find what we need in the existing [ContactsBookViewModel](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/contactsbook/ContactsBookViewModel.kt#L43)
      - [x] the contacts from direct chats (see [getContactsFromDirectChats()](https://github.com/dinsic-pim/tchap-android/blob/develop/vector/src/main/java/fr/gouv/tchap/util/DinsicUtils.java#L1045, Note: this function is doing more than just retrieve some contacts, indeed it updates the direct dictionary by replacing some keys (when these keys are e-mail). This email keys are added when the user invite someone by their email address. We will handle this in another issue)
   - [x] second section "Répertoire Tchap". This section is displayed only in case of search. It lists the result of the search request in the users directory. The search in the users directory is not allowed for an external account (see [tchapv1](https://github.com/dinsic-pim/tchap-android/blob/develop/vector/src/main/java/fr/gouv/tchap/fragments/TchapContactFragment.java#L250)), we will consider the external account case later. 
Note: The search itself depends on #52, I suggest preparing here a method which will be ready to be used when the search will be implemented.

The second tab may be built by using the current activity named [CreateDirectRoomActivity](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomActivity.kt) in Element-Android

Note: the items of these tab will be displayed like in Element-Android. We will not apply the Tchap design on them for the moment.

The navigation bar will be handled by #52

Note2: We will keep the floating button (displayed over the list at the bottom right), we will update it later

**Comment by giomfo** [comment]
Selected for sp43 for the contacts lists
